### PR TITLE
Feature/tda 180

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: dags/etl_executor.py [hunk 1]
```diff
@@ -28,10 +29,26 @@ def init_etl_pipelines_register():
...
+
+    tables_metadata = config.TABLES_METADATA
+
+    for table_name, query_path in config.TRIPLE_STORE_TABLE_QUERY_PATHS.items():
         etl_pipelines_register.register(etl_pipeline_name=table_name,
                                         etl_pipeline=PostgresETLPipeline(table_name=table_name,
-                                                                         sparql_query_path=query_path, primary_key_column_name=f"{table_name}Id"))
+                                                                         sparql_query_path=query_path,
+                                                                         primary_key_column_name=
+                                                                         tables_metadata[table_name]['PK'],
```
Comment (CaptainOfHacks) [inline dags/etl_executor.py:40]: replace strings with constants

File: dags/etl_executor.py [hunk 2]
```diff
@@ -28,10 +29,26 @@ def init_etl_pipelines_register():
...
+                                                                         foreign_key_column_names=
+                                                                         tables_metadata[table_name]['FK']))
```
Comment (CaptainOfHacks) [inline dags/etl_executor.py:42]: replace strings with constants

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
We need to discuss about date range and replace strings with constants.

File: dags/run_etl_executor_by_date_range.py [hunk 3]
```diff
@@ -63,8 +63,8 @@ def run_etl_executor_for_each_date_in_range():
                                         END_DATE_METADATA_FIELD: premium_index_start_date},
                                    }
             postgres_table_names = {
-                table_name: {START_DATE_METADATA_FIELD: start_date, END_DATE_METADATA_FIELD: end_date} for table_name in
-                config.TABLE_QUERY_PATHS.keys()}
+                table_name: {START_DATE_METADATA_FIELD: date, END_DATE_METADATA_FIELD: date} for table_name in
```
[the remark this answers is not in the document]
Comment (duprijil) [reply dags/run_etl_executor_by_date_range.py:66]: This DAG creates ETL executor for a day in date range, firstly when I added start_date and end_date was wrong. Now logic is correct.

[the remark this answers is not in the document]
Comment (duprijil) [reply dags/run_etl_executor_by_date_range.py:66]: Now this DAG works with date range but correctly

**Review by CaptainOfHacks — APPROVED** [review]

**Pull request merged**